packages/arm-core/src/components/MapOverlay.tsx

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components'
+
+const MapOverlay = styled.div`
+  display: flex;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  top: 0;
+  z-index: 1;
```

Comment:
@jonkoops @chrisvanmook  This z-index seems to be causing the panels not to show up above the map on the Mobile story